stories/grid.stories.js

```diff
@@ -1,44 +1,126 @@
 import notes from './grid.md';
-import { Box, Grid } from '../src';
+import { Box, Grid, Text } from '../src';
 
 export default {
   title: 'Grid'
 };
 
+export const Default = () => (
+  <>
+    <Text mb="small">
+      Grids are used for organizing components on a page. It is essentially a
+      Box with extra features
+    </Text>
+    <Grid>
+      <Box p="regular" backgroundColor="blue" />
+      <Box p="regular" backgroundColor="red" />
+      <Box p="regular" backgroundColor="blue" />
+    </Grid>
+  </>
+);
+
+Default.storyName = 'default';
+Default.parameters = { notes: { markdown: notes } };
+
 export const GridGap = () => {
   return (
-    <Grid gridGap={['smaller', 'regular', 'larger']}>
-      <Box p="regular" backgroundColor="blue">
-        1
-      </Box>
-      <Box p="regular" backgroundColor="red">
-        2
-      </Box>
-      <Box p="regular" backgroundColor="blue">
-        3
-      </Box>
-    </Grid>
+    <>
+      <Text>gridGap sets the gap size between Grid child components</Text>
```

Comment:
I think it's worth linking out to https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout as well, since these props are just a layer over the CSS properties.